CryptoPrices/Sources/Home/Tests/HomeTests/HomeViewModelSpec.swift

```diff
@@ -0,0 +1,65 @@
+// swiftlint:disable closure_body_length
+import DomainTestHelpers
+import Nimble
+import Quick
+import TestHelpers
+@testable import Home
+
+final class HomeViewModelSpec: QuickSpec {
+
+    override func spec() {
+        var homeViewModel: HomeViewModel!
+        var myCoinsUseCase: MockMyCoinsUseCase!
+
+        describe("the HomeViewModel") {
+            beforeEach {
+                myCoinsUseCase = MockMyCoinsUseCase()
+                homeViewModel = await HomeViewModel(myCoinsUseCase: myCoinsUseCase)
+            }
+
+            describe("its initial state") {
+                it("has the correct value for myCoins") {
+                    await expect {
+                        await homeViewModel.myCoins
+                    }
+                    .to(beEmpty())
+                }
+            }
+
+            describe("its fetchMyCoins()") {
```

Comment:
Just a small refactor to reflect our iOS team's latest discussion regarding the convention here: https://github.com/nimblehq/compass/discussions/1048 
```suggestion
            describe("its fetchMyCoins() call") {
```